### PR TITLE
Added configurable border width and disable flag for the shadow on tapping (helpful if you want a flat button and the elevation is 0)

### PR DESCRIPTION
**Pull request by JaseElder**
Added configurable border width and disable flag for the shadow on tapping (helpful if you want a flat button and the elevation is 0)

### CODE REVIEW

**Review by Frezyx — CHANGES_REQUESTED** [review]
Hello @JaseElder ! 
Thanks a lot for this contribution 

Can we fix review issues before merge ?

File: lib/src/group_button_body.dart [hunk 1]
```diff
@@ -59,8 +61,10 @@ class GroupButtonBody<T> extends StatefulWidget {
   final Color? selectedBorderColor;
   final Color? unselectedBorderColor;
   final BorderRadius? borderRadius;
+  final double? borderWidth;
   final List<BoxShadow> selectedShadow;
   final List<BoxShadow> unselectedShadow;
+  final bool? disableTappingShadow;
```
[the remark this answers is not in the document]
Comment (JaseElder) [reply lib/src/group_button_body.dart:67]: Yes, good idea! I'll amend.

**Comment by JaseElder** [comment]
I'm not sure whether I've done the right thing with this PR, but it looks like the new changes are there...

**Comment by jenyang2** [comment]
When does border with apply?

**Comment by JaseElder** [comment]
> When does border width apply?

Border width is by default set to 1, but a borderWidth can be provided in GroupButtonOptions() if it needs to be other than 1.

**Comment by Nirsu** [comment]
@Frezyx Any news on this PR ?

File: lib/src/options/group_button_options.dart [hunk 2]
```diff
@@ -17,8 +16,10 @@ class GroupButtonOptions {
     this.selectedBorderColor,
     this.unselectedBorderColor,
     this.borderRadius,
+    this.borderWidth,
     this.selectedShadow = defaultShadow,
     this.unselectedShadow = defaultShadow,
+    this.tappingShadowColor = defaultTappingShadowColor,
```
Comment (Frezyx) [inline lib/src/options/group_button_options.dart:22]: tappingShadowColor must be null by default

**Review by Frezyx — CHANGES_REQUESTED** [review]

**Comment by Frezyx** [comment]
Hello @JaseElder !
Thank you for this contribution ❤️ 

Sorry for delay 🥶

**Review by Frezyx — APPROVED** [review]

**Comment by JaseElder** [comment]
> Hello @JaseElder ! Thank you for this contribution ❤️
> 
> Sorry for delay 🥶

@Frezyx No worries!

Is there something special that has to happen to have this release appear on pub.dev? It's still showing group_button: ^5.3.4 for me

**Comment by JaseElder** [comment]
> > Hello @JaseElder ! Thank you for this contribution ❤️
> > Sorry for delay 🥶
> 
> @Frezyx No worries!
> 
> Is there something special that has to happen to have this release appear on pub.dev? It's still showing group_button: ^5.3.4 for me

Hi @Frezyx I'm still seeing 5.3.4 on pub.dev. Is it possible to push this release up?

**Comment by JaseElder** [comment]
Hi @Frezyx ,

Just doing some admin and noticed that 5.4.0 is still not published on pub.dev. Is it possible for you to push it up?
Thanks.

**Pull request merged**